mailpoet/lib/Util/Notices/DeprecatedFilterNotice.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace MailPoet\Util\Notices;
+
+use MailPoet\Util\Helpers;
+use MailPoet\WP\Functions as WPFunctions;
+use MailPoet\WP\Notice;
+
+/**
+ * This can be removed after 2022-12-01
+ */
+class DeprecatedFilterNotice {
```

Comment:
Why not using `$this->wp->deprecatedHook`?

Reply:
There are three main reasons:
1. If I am not wrong `deprecatedHook` works only when the hook is called.
2. We don't have anymore a class from Swift Mailer that is necessary as instance in filter parameter.
3. The filter is called in a background job, so a customer shouldn't notice